packages/engine/src/datastore/batch/mod.rs

```diff
@@ -9,10 +9,10 @@ pub mod message;
 pub mod metaversion;
 pub mod migration;
 
-pub use agent::Batch as AgentBatch;
-pub use context::{AgentIndex, Batch as ContextBatch, MessageIndex};
-pub use dataset::Batch as Dataset;
-pub use message::Batch as MessageBatch;
+pub use agent::AgentBatch;
+pub use context::{AgentIndex, ContextBatch, MessageIndex};
+pub use dataset::Dataset;
+pub use message::MessageBatch;
```

Comment:
This is the module, where all the batches got aliased to the type. IDE fails to interpret `Batch` as e.g. `AgentBatch` at this point. Changes in this PR are to remove `Batch as`.